<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spasma_Mode</class>
 <widget class="QMainWindow" name="Spasma_Mode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Data Analysis Main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Choose data</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTextBrowser" name="text_browser_file_display">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_add_files">
           <property name="text">
            <string>add files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_reset_files">
           <property name="text">
            <string>reset files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_load">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>load</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Analyze data</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTableWidget" name="Table_LAP">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="rowCount">
            <number>3</number>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>Loader</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Analyzer</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Plotter</string>
            </property>
           </row>
           <column/>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_settings">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>open settings window</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_analyze">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>analyze</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_plot">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>open plot window</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_replot_reanalyze">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>re-analyze and re-plot</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save_Settings"/>
    <addaction name="action_Load_Settings"/>
    <addaction name="separator"/>
    <addaction name="action_Choose_Loader"/>
    <addaction name="action_Choose_Analyzer"/>
    <addaction name="action_Choose_Plotter"/>
    <addaction name="separator"/>
    <addaction name="action_Reload_Modules"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_Save_Settings">
   <property name="text">
    <string>&amp;Save Settings</string>
   </property>
  </action>
  <action name="action_Load_Settings">
   <property name="text">
    <string>&amp;Load Settings</string>
   </property>
  </action>
  <action name="action_Choose_Loader">
   <property name="text">
    <string>&amp;Choose Loader</string>
   </property>
  </action>
  <action name="action_Choose_Analyzer">
   <property name="text">
    <string>&amp;Choose Analyzer</string>
   </property>
  </action>
  <action name="action_Choose_Plotter">
   <property name="text">
    <string>&amp;Choose Plotter</string>
   </property>
  </action>
  <action name="action_Reload_Modules">
   <property name="text">
    <string>&amp;Reload Modules</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
